src/Cache/FilesystemCache.php

```diff
@@ -68,7 +68,7 @@ public function write($key, $content)
             if (self::FORCE_BYTECODE_INVALIDATION == ($this->options & self::FORCE_BYTECODE_INVALIDATION)) {
                 // Compile cached file into bytecode cache
                 if (\function_exists('opcache_invalidate') && filter_var(ini_get('opcache.enable'), FILTER_VALIDATE_BOOLEAN)) {
-                    opcache_invalidate($key, true);
+                    @opcache_invalidate($key, true);
```

Comment:
So, you're saying that the method is available, that's opcache is enabled, but some hosters disable the function somehow?

Reply:
The method will work but it will just return that it is disabled for security reasons, using the opcache API you can enumerate all files on the host even those that are not owned. Apnscp does this by default.